algorithms/refinement/reflection_manager.py

```diff
@@ -314,14 +318,21 @@ def from_parameters_reflections_experiments(
                 *params.weighting_strategy.constants, stills=do_stills
             )
 
+        # Check for deprecated parameter
+        if params.trim_scan_edges is not None:
+            logger.warn(
```

Comment:
`logger.warn` is [deprecated](https://docs.python.org/3.6/library/logging.html#logging.Logger.warning)
```suggestion
            logger.warning(
```

Reply:
Although I wonder if as in conversation above, `FutureWarning` would be the more correct way. Did this not show to the user?

Reply:
Oh I somehow missed that above. Yes, it does show to the user, but it is not highlighted in yellow like `logger.warning` is.